lib/galaxy/model/__init__.py

```diff
@@ -11569,6 +11569,73 @@ def __repr__(self):
         )
 
 
+class UserCredentials(Base):
+    """
+    Represents a credential associated with a user for a specific service.
+    """
+
+    __tablename__ = "user_credentials"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    user_id: Mapped[int] = mapped_column(ForeignKey("galaxy_user.id"), index=True, nullable=False)
```

Comment:
`nullable=False` is not needed: `NOT NULL` is inferred from the typing hint (which is not `Optional[int]`)
Same goes for all other instances of `nullable=False` in all other model definitions.